### PR TITLE
Fixing broken link in CaaS blog

### PR DESCRIPTION
**Pull request by anuragpaliwal80**
Fixing broken link in CaaS blog

### CODE REVIEW

File: blog/2021-06-25-chaos-mesh-remake-one-step-closer-toward-chaos-as-a-service.md [hunk 1]
```diff
@@ -109,40 +109,40 @@ A closed loop of Chaos Engineering includes four steps: exploring chaos, discove
...
 
 * Develop a controller or operator to manage CRDs.
 * Handle CRD events uniformly and operate on CRDs via HTTP callback. This method only uses HTTP APIs, with no requirement on Golang. For an example, see [Whitebox Controller](https://github.com/summerwind/whitebox-controller).
-* Use WebAssembly (Wasm). When you need to call chaos experiment logic, just call the Wasm program. See Vector's [WASM Transform](https://vector.dev/docs/reference/transforms/wasm/).
-* Use SQL to query the chaos experiment status. Because Chaos Mesh is based on CRDs, you can use SQL to operate on Kubernetes. Examples include [Presto connector](https://github.com/xuxinkun/kubesql) and [osquery extension](https://github.com/aquasecurity/kube-query). 
+* Use WebAssembly (Wasm). When you need to call chaos experiment logic, just call the Wasm program. See Vector's [WASM Transform](https://webcache.googleusercontent.com/search?q=cache:aR1ZIlTwz9sJ:https://vector.dev/docs/r[…]uration/transforms/wasm/+&cd=1&hl=zh-CN&ct=clnk&gl=jp).
```
Comment (milasuperstar) [inline blog/2021-06-25-chaos-mesh-remake-one-step-closer-toward-chaos-as-a-service.md:137]:
```suggestion
* Use WebAssembly (Wasm). When you need to call chaos experiment logic, just call the Wasm program. See Vector's [WASM Transform](https://vector.dev/docs/reference/transforms/wasm/).

```

Comment (milasuperstar) [reply blog/2021-06-25-chaos-mesh-remake-one-step-closer-toward-chaos-as-a-service.md:137]: Just checked! The original link works again now, for some reason...

**Comment by en-jin19** [comment]
Hi @anuragpaliwal80, thank you for your comments! We have checked that the former broken link is fixed and available to visit. Could we only keep the changes of removing extra blanks and undo the link modification in this PR?

**Comment by anuragpaliwal80** [comment]
As it's working, I am closing this PR.

**Pull request closed without merging**